src/nl/release-notes/20.frontend.md

```diff
@@ -0,0 +1,28 @@
+---
+title: MyParcel UI notes
```

Comment:
"Wat is een UI?"

Reply:
Ons beoogde publiek met deze notes bestaat uit developers, die weten toch wel wat een UI is?

Reply:
blijkbaar heb ik dat verkeerd begrepen 🤔 maar is "frontend" dan beter? Mitchell houdt "frontend" en "backoffice" en "frontoffice" en zelfs "backend" zelf niet altijd uit elkaar namelijk. En "website" oid lijkt me te breed